cmd/lintroller/lintroller.go

```diff
@@ -76,6 +77,19 @@ func main() {
 			}
 		}
 
+		warnTable := []struct {
+			Warn     bool
+			Analyzer *analysis.Analyzer
+		}{
+			{cfg.Errorlint.Warn, &errorlint.Analyzer},
+		}
+
+		for i := range warnTable {
+			if warnTable[i].Warn {
+				analyzers = append(analyzers, table[i].Analyzer)
+			}
+		}
```

Comment:
The anonymous struct above can just be modified to take `Warn` as a boolean instead of creating a new one here.